tests/projects/test_projects.py

```diff
@@ -45,17 +45,45 @@ def local_git_repo_uri(local_git_repo):
     return "file://%s" % local_git_repo
 
 
-def test_fetch_project(local_git_repo, local_git_repo_uri):
+@pytest.fixture()
+def zipped_repo(tmpdir):
+    import zipfile
+    zip_name = tmpdir.join('%s.zip' % TEST_PROJECT_NAME).strpath
+    with zipfile.ZipFile(zip_name, 'w', zipfile.ZIP_DEFLATED) as zip_file:
+        for root, _, files in os.walk(TEST_PROJECT_DIR):
+            for file_name in files:
+                file_path = os.path.join(root, file_name)
+                zip_file.write(file_path, file_path[len(TEST_PROJECT_DIR)+len(os.sep):])
+    return zip_name
+
+
+def test_is_zip_uri():
```

Comment:
Nice! Can we also add some tests here to ensure that uri's which are *not* zip uri's are classified correctly?